oap-server/server-starter/src/main/resources/otel-oc-rules/vm.yaml

```diff
@@ -28,6 +28,7 @@
 #    "-P6H3M"    -- parses as "-6 hours and -3 minutes"
 #    "-P-6H+3M"  -- parses as "+6 hours and -3 minutes"
 # </pre>
+filter: "{ tags -> tags.job_name == 'vm-monitoring' }"
```

Comment:
I found `oap.yaml` has the same metrics with `istio-controlplane.yaml`, not `vm`
```
process_cpu_seconds_total
```

Reply:
> I found `oap.yaml` has the same metrics with `istio-controlplane.yaml`, not `vm`
> 
> ```
> process_cpu_seconds_total
> ```

There are metrics name conflicts in `vm` and `so11y` for sure, you cannot only compare our configuration files, the key is what metrics actually OpenTelemetry sends. FYI the conflict between `so11y` and `vm` is `instance_cpu_percentage` from what I can tell.

Reply:
I think we should recommend(if not require) to add filter for all oc-receiver MAL. In the future, OpenTelemetry native metric format(just GA) and meter receiver should recommend this setup too.